src/main/java/levy/art/RijkService.java

```diff
@@ -1,11 +1,14 @@
 package levy.art;
 
+import com.andrewoid.ApiKey;
 import io.reactivex.rxjava3.core.Single;
 import levy.art.json.RijksCollection;
 import retrofit2.http.GET;
 import retrofit2.http.Query;
 
 public interface RijkService {
+    ApiKey apiKey = new ApiKey();
+    String keyString = apiKey.get();
```

Comment:
I did not say put this in your interface.

Reply:
so where should i put

Reply:
Back where it was. But only do it once, not every time you search.

Reply:
so what else what i need besides the beforeall?

Reply:
I never said anything about beforeall

Reply:
so how should i do it

Reply:
Not sure how to do that